app/controllers/discussions_controller.rb

```diff
@@ -0,0 +1,24 @@
+# Main site controller for discussions and comments
+class DiscussionsController < ApplicationController
+  before_action :check_feature_flags
+
+  def index
+    # TODO: list of recently active discussions etc
+    redirect_to root_path
+  end
+
+  def show
+    @discussion = Discussion.find( params[ :id ] )
```

Comment:
Could this allow a user to view a hidden discussion? If yes, consider scoping to published discussions. This might be a good spot to use pundit's `policy_scope`.